node/src/main/scala/org/bitcoins/node/networking/peer/PeerMessageSender.scala

```diff
@@ -72,7 +72,10 @@ case class PeerMessageSender(client: P2PClient)(implicit conf: NodeAppConfig)
 
   def sendVersionMessage(chainApi: ChainApi)(implicit
       ec: ExecutionContext): Future[Unit] = {
-    chainApi.getBestHashBlockHeight().flatMap { height =>
+    val heightF =
```

Comment:
Hmm... what is this for ? Why is this needed now?

Reply:
Ran into this with some of the testing stuff, basically, if our db is empty then we should return a `0`

Reply:
Ok here is a patch to fix this so we can remove this: 

https://github.com/Christewart/bitcoin-s-core/commit/69fa407435501bd0f1e4cf8d585c684f37608a0b